docs/benchmarks/model-benchmarks.md

```diff
@@ -0,0 +1,113 @@
+---
+id: model-benchmarks
+---
+
+# Model Benchmarks
+
+## Benchmarking list
+
+### GPT-2 models
+
+### BERT models
+
+### LSTM models
+
+### CNN models
+
+
+## Benchmarking metrics
+
+<table>
+  <tbody>
+    <tr align="center" valign="bottom">
+      <td>
+      </td>
+      <td>
+        <b>Micro Benchmark</b>
```

Comment:
why don't remove 'Micro Benchmark'?

Reply:
it's placeholder, to be updated by tingting